### ISSUE TITLE
.NET 4.* tests raises EXCEPTION_ACCESS_VIOLATION within the JVM

### ISSUE THREAD

**Issue by masesdevelopers** [masesgroup/JCOReflector#112]
**Describe the bug**
When the test code runs, within the GitHub environment, against .NET 4.* environment some of them fails.

**To Reproduce**
1. Run the testbranch.yaml
2. Wait execution of .NET 4.* tests
3. See if results contains errors
4. Download the **Errors** artifacts to get logs and dumps

**Expected behavior**
The tests shall execute without problems like the same executed in .NET Core 3.1 and .NET 6.

**Screenshots**
![image](https://user-images.githubusercontent.com/94312179/185494101-f7f0a925-fd4d-4de6-b0d8-d325c71482fd.png)

**Desktop (please complete the following information):**
 - OS: GitHub virtual environment 
 - Image Version: 20220816.1

**Additional context**
The tests which fails in GitHub environment runs without problems in local test environment.

**Comment by masesdevelopers** [comment]
@mariomastrodicasa probably the problem is related to something was changed in the Image version of GitHub Virtual environment. We opened another issue (https://github.com/masesgroup/JNet/issues/35) and seems related to the update of Visual Studio to the latest version in the image 20220816.1. Locally the failed tests runs without problem.
We changed the windows.yaml workflow to continue in case of error and go over with update of JCOReflector, meanwhile we are trying to analyze the dumps generated from the testbranch.yaml runs.
